src/main/java/org/gridsuite/modification/server/dto/ModificationInfos.java

```diff
@@ -83,6 +83,12 @@ public class ModificationInfos {
     @Schema(description = "Modification flag")
     private Boolean stashed;
 
+    @Schema(description = "Message type")
+    private String messageType;
```

Comment:
What does `message` prefix convey ? I'm a bit lost since here it's about modifications